src/overlays/actors/ovl_En_Fg/z_en_fg.c

```diff
@@ -9,7 +15,29 @@ void EnFg_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnFg_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnFg_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnFg_Jump(EnFg* this, GlobalContext* globalCtx);
+void EnFg_DoNothing(EnFg* this, GlobalContext* globalCtx);
+void EnFg_Knockback(EnFg* this, GlobalContext* globalCtx);
+void EnFg_AddDust(EnFgEffectDust* dustEffect, Vec3f* worldPos);
+void EnFg_UpdateDust(EnFgEffectDust* dustEffect);
+void EnFg_DrawDust(GlobalContext* globalCtx, EnFgEffectDust* dustEffect);
+
+extern u64 gDust1Tex[]; // D_0408DBE0
```

Comment:
Let's go ahead and remove the names. Feel free to document them, but we want to keep the object, gameplay keep, and dangeon/field keep segments clean until we're ready to start object decomp. These names will be added then.